utils/functions

```diff
@@ -33,6 +33,13 @@
                  34sns-repair-1n-1f
                  35m0singlenode
                  36spare-reservation
+# 37protocol ST checks checksum of the BE structures during
+# developmental phase they are bound to fail so code
+# changes are required but ST has to be in disabled
+# state for now when the release is in GA phase we will
+# enable the ST again to ensure there will be no changes
+# in BE structures.
```

Comment:
```suggestion
# 37protocol ST validates checksum of the BE structures, during
# development phase these checks are bound to fail. 
# Hence ST has to be in disabled till GA release phase.
# We can re-enable this ST again after GA release
# to ensure that there are no changes in BE structures.
```

Please replace same comment in above change as well.
@shashank-parulekar please correct here if something is missing.